sdcm/cluster.py

```diff
@@ -3437,7 +3437,13 @@ def node_setup(self, node, verbose=False, timeout=3600):
             node.remoter.run('sudo cat /etc/scylla.d/io.conf')
 
             if self.params.get('use_mgmt', None):
-                region = self.params.get('region_name').split()
+                cluster_backend = self.params.get("cluster_backend")
+                if cluster_backend == "aws":
+                    region = self.params.get('region_name').split()
+                elif cluster_backend == "gce":
+                    region = self.params.get('gce_datacenter').split()
+                else:
+                    raise Exception(f"Unknown cluster backend: '{cluster_backend}'")
```

Comment:
what about 'aws-siren' backend?

Reply:
in aws-siren we are not installing manager, so this is fine.

Reply:
as Israel said we don't support any cloud for storing backups other than AWS S3